.docker/php/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-ARG PHP_VERSION=8.2-cli
+ARG PHP_VERSION=8.4-cli
```

Comment:
Could you please perform a rebase?

Reply:
`git pull --rebase upstream main && git push --force origin main` should be enough to remove any "duplication" between the changes introduced in this PR, where `upstream` is the remote pointing to https://github.com/doctrine-extensions/DoctrineExtensions and `origin` the remote pointing to https://github.com/KeeSystemOrg/DoctrineExtensions.

Reply:
Done, the rebase was not enough to fix, but I amended the duplicated commit that indeed contained my changes and force-pushed it. Now it's feb082ce. Thanks!